client/state/data-layer/wpcom/sites/atomic/transfers/index.js

```diff
@@ -8,17 +8,41 @@ import { registerHandlers } from 'calypso/state/data-layer/handler-registry';
 import { http } from 'calypso/state/data-layer/wpcom-http/actions';
 import { dispatchRequest } from 'calypso/state/data-layer/wpcom-http/utils';
 
+export const mapToRequestBody = ( action ) => {
+	const requestBody = {};
+
+	if ( action.softwareSet ) {
+		requestBody.software_set = action.softwareSet;
+	}
+
+	if ( action.themeSlug ) {
+		requestBody.theme_slug = action.themeSlug;
+	}
+
+	if ( action.pluginSlug ) {
+		requestBody.plugin_slug = action.pluginSlug;
+	}
+
+	if ( action.themeFile ) {
+		requestBody.theme_file = action.themeFile;
+	}
+
+	if ( action.pluginFile ) {
+		requestBody.plugin_file = action.pluginFile;
+	}
+
+	return requestBody;
+};
+
 const initiateAtomicTransfer = ( action ) =>
 	http(
 		{
 			apiNamespace: 'wpcom/v2',
 			method: 'POST',
 			path: `/sites/${ action.siteId }/atomic/transfers/`,
-			...( action.softwareSet
+			...( action.softwareSet || action.themeSlug
```

Comment:
Shouldn't we allow `action.pluginSlug` here as well?

Reply:
We use a different method for atomic transformation with plugins, so the plugin slug is not needed here.

https://github.com/Automattic/wp-calypso/blob/50f6cee2b3db04d0c300fc4ff970225fade2f2c5/client/my-sites/marketplace/pages/marketplace-plugin-install/index.tsx#L215-L220